omnipaxos_storage/src/memory.rs

```diff
@@ -1,239 +1,377 @@
 #[allow(missing_docs)]
 pub mod persistent_storage {
```

Comment:
This module is a bit misplaced, it is persistent but is under `memory`.

I would suggest either renaming this file to something more generic e.g. `storage.rs` or separating the memory and persistent storage modules into two different files.

Reply:
`memory.rs` can be split into `persistent_storage.rs` and `memory_storage.rs`. PersistentStorage has grown large due to the wrapper structs used for storage and the new PersistentStorageConfig struct that were added.

Reply:
That is fine with me.